### DISCUSSION TITLE
Welcome to Google Sites Themes!

### DISCUSSION THREAD

**Discussion by Narlai** [iindicator/Google-Sites-Themes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ilsicca** [comment]
Hi @iindicator ! How are you? I'm interested in your Google Sites Themes. How I can contact you?

Regards!

**Reply by Narlai** [reply]
Which one do you like? Send me your email and I will share it with you.

**Reply by ilsicca** [reply]
Hi! Thanks for your reply! I interested on two templates: 

- Bakery: [https://sites.google.com/view/big-template-bakery/home](url)
- Web Design: [https://sites.google.com/view/bog-template-web-design/home](url)

My email: ignaciolsiccardi@gmail.com

Thank you so much!
